client/landing/stepper/declarative-flow/internals/analytics/record-design.ts

```diff
@@ -50,9 +51,10 @@ export function recordSelectedDesign( {
 	}
 }
 
-export function getDesignTypeProps( design?: Design ) {
+export function getDesignTypeProps( design: Design ) {
 	return {
 		goes_to_assembler_step: design?.design_type === 'assembler',
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

Maybe we can remove `?`?
```suggestion
		goes_to_assembler_step: design.design_type === 'assembler',
```